README.md

```diff
@@ -65,3 +65,5 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
 The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
 
 Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+
+[![https://vercel.com?utm_source=my-covid-story&utm_campaign=oss](./public/powered-by-vercel.svg)](https://vercel.com?utm_source=my-covid-story&utm_campaign=oss)
```

Comment:
Do we also want to/need to add in the GitHub sponsor file for Vercel?

Reply:
@JamiesonRoberts not sure I follow?

Reply:
On second thought, I think I'm a dummy, and it can't be used for that! So its all good!